### ISSUE TITLE
[Feature Request] Raze support

### ISSUE THREAD

**Issue by NightFright2k19** [Youda008/DoomRunner#14]
I know Raze is still in early development, but it would be very convenient to be able to use this launcher with Raze games (Duke3D, Blood etc) as well. I guess quite some adjustments would need to be made, but hopefully it won't be necessary to create a separate project for this. Ideally, if Raze works, it could be adjusted for EDuke32 as well, if any modifications are required.

**Comment by Youda008** [comment]
Hm, a fork of Eduke32? I started experimenting with Eduke32 only short ago, so it's too early to say, but this sounds like an overkill. This launcher was meant for ZDoom, it doesn't even support non-ZDoom Doom engines like Doomsday yet. And i doubt that EDuke's command line interface is any simmilar to ZDoom ones.
Honestly, making any cross-game launcher is a big challange, because those games and their system of loading maps and mods can wildly differ. It would be difficult to even design the graphical user interface to be universal for multiple games and yet still powerfull in what it can do and how easy it is to use it.
For example the second tab of DoomRunner is totaly ZDoom specific, it does not make much sense for DukeNukem to set "fast monsters" and "compatibility options", there aren't such things.

If a design of an application could be expressed by a triangle, it would look like this, and the DoomRunner is trying to be in the red dot
<img src="https://user-images.githubusercontent.com/20306543/75458980-005c7780-597f-11ea-9acd-4ab0670405cf.png"  width="500" height="300" />
In other words: You can't have everything.

**Comment by Youda008** [comment]
https://wiki.eduke32.com/wiki/Command_line_options

**Comment by Youda008** [comment]
https://wiki.eduke32.com/wiki/Command_line_options
https://zdoom.org/wiki/Command_line_parameters
That really has almost nothing in common. The launcher would become seriously schizophrenic.

**Comment by NightFright2k19** [comment]
Well, I didn't expect everything to fit into the same launcher, anyway. But if a separate one can be made which uses EDuke32 commands, it'd be amazing. The name "DoomRunner" wouldn't fit otherwise, anyway,
